src/silx/app/view/test/test_plotselection.py

```diff
@@ -0,0 +1,234 @@
+import pytest
+from silx.gui import qt
+from silx.io.dictdump import dicttoh5
+import silx.io
+from silx.app.view.CustomPlotSelectionWindow import CustomPlotSelectionWindow
+
+
+@pytest.fixture
+def setupMimeData(tmpdir):
```

Comment:
This can be renamed to what it provides instead of what it does, e.g. :
```suggestion
def silxMimeData(tmpdir):
```